metadata_writer_sqlite.h

```diff
@@ -73,7 +76,10 @@
                             "Sequence INTEGER NOT NULL," \
                             "L3SessionId INTEGER NOT NULL," \
                             "L4SessionId INTEGER NOT NULL DEFAULT 0," \
-                            "EventValueStr TEXT NOT NULL," \
+                            "HasIp INTEGER," \
+                            "Connectivity INTEGER," \
+                            "ConnectionMode INTEGER," \
+                            "Quality INTEGER," \
```

Comment:
Please ask Tobias/Jonas about this. I am not sure how any non-upgraded DB will behave or if all DBs will be updated. I think that when in SQL mode, we should create the exact same SQL as before.